src/status_im/multiaccounts/login/core.cljs

```diff
@@ -476,12 +476,23 @@
         keychain/auth-method-biometric
         keychain/auth-method-password))))
 
+(rf/defn local-pairing-completed
+  {:events [:local-pairing/completed]}
+  [{:keys [db] :as cofx}]
+  (rf/merge cofx
+            {:db       (dissoc db :local-pairing/completed-pairing?)
+             :dispatch [:init-root :enable-notifications]}))
+
 (defn redirect-to-root
   "Decides which root should be initialised depending on user and app state"
   [db]
-  (if (get db :tos/accepted?)
-    (re-frame/dispatch [:init-root :shell-stack])
-    (re-frame/dispatch [:init-root :tos])))
+  (let [tos-accepted?            (get db :tos/accepted?)
```

Comment:
do we have :tos/accepted in the new flows? as far as I can see we don't request this anymore.
cc @cammellos

Reply:
I think it could be removed, I'd do it in the scope of a different PR though, just to make sure :)